spec/support/test_helpers.rb

```diff
@@ -14,13 +14,13 @@ module TestHelpers
 
   shared_context 'claim-types helpers' do
     let(:agfs_claim_types) { %w[agfs agfs_interim agfs_supplementary agfs_hardship] }
-    let(:lgfs_claim_types) { %w[lgfs_final lgfs_interim lgfs_transfer] }
+    let(:lgfs_claim_types) { %w[lgfs_final lgfs_interim lgfs_transfer lgfs_hardship] }
     let(:all_claim_types) { agfs_claim_types | lgfs_claim_types }
   end
 
   shared_context 'claim-types object helpers' do
     let(:agfs_claim_object_types) { %w[Claim::AdvocateClaim Claim::AdvocateInterimClaim Claim::AdvocateSupplementaryClaim Claim::AdvocateHardshipClaim] }
-    let(:lgfs_claim_object_types) { %w[Claim::LitigatorClaim Claim::InterimClaim Claim::TransferClaim] }
+    let(:lgfs_claim_object_types) { %w[Claim::LitigatorClaim Claim::InterimClaim Claim::TransferClaim Claim::LitigatorHardshipClaim] }
```

Comment:
going to need rebasing as I moved this to `spec/support/shared_examples_for_claim_types.rb`